src/androidTest/java/de/dennisguse/opentracks/content/provider/CustomContentProviderUtilsTest.java

```diff
@@ -75,6 +77,17 @@ public void setUp() {
         contentProviderUtils.deleteAllTracks(context);
     }
 
+    private void cleanPhotoDir(File file) {
+        if (file != null && file.exists() && file.isDirectory()) {
```

Comment:
if (file == null) {
  return;
}

Reply:
I've removed this code because it's not needed.